### DISCUSSION TITLE
Taskbar hovering names and right clicking desktop options

### DISCUSSION THREAD

**Discussion by Mandingalator** [valinet/ExplorerPatcher#764] [Q&A]
Basic Info:
Windows 11 Pro Version 21H2 OS build 22000.469
ExplorerPatcher Version 22000.469.41.16

Taskbar Hovering Names:
I am wanting to turn off the pop-up label/name of icons when hovering over them. For example, when hovering over the volume button on the bottom right of the taskbar the name pops up above it. I want to remove this completely if possible. Here is an example of the text I am talking about. 

![1](https://user-images.githubusercontent.com/98803237/151938410-637c64db-703e-4111-a1a1-13266e4c1381.png)
 
Right Click Desktop Options:
After a few hours of reading and tinkering with ExplorerPatcher I noticed some of my options are missing when right clicking the desktop, specifically my GeForce Experience option. Nvidia control panel is still there, but not GeForce Experience. This is not a huge deal, but my main question is if it is possible to customize the list of options that pop up and be able to add/remove programs from the list? 

I am new to ExplorerPatcher, and honestly quite new to customizing options in Windows all together besides basics. I read through the features page completely, and the frequently asked questions page is down, hopefully these questions have not been asked before. Any help would be great, thanks.

**Comment by valinet** [comment]
EP doesn’t alter the context menus. What misses or is there is because of some other reason.

Removing labels for system tray icons is not a feature at the moment.

**Reply by Mandingalator** [reply]
Thanks for the quick reply and help. Is there a way to remove the labels for tray icons at all?
